### PR TITLE
Sch tests 140

### PR DESCRIPTION
**Pull request by shewood**
## [ADO-140348](https://dev.azure.com/VP-BD/DECD/_workitems/edit/140348)

Fixed [AB#140348](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/140348)

### Description of proposed changes:
Updated some of the tests to fix the console errors for act and undefined
Removed the dts-stn/service-canada-design-system from application. 

### What to test for/How to test
npm i
run unit tests and look at the console log. 
Application continues to work without the DS. 

### Additional Notes
Still TODO:  logout.test

### CODE REVIEW

File: __tests__/components/ProfileTasks.test.js [hunk 1]
```diff
@@ -14,11 +14,15 @@ describe('ProfileTasks', () => {
           title: 'Province of residence',
           icon: 'circle',
           link: '/dashboard',
+          areaLabel: 'taks1',
```
Comment (krischarbonneau) [inline __tests__/components/ProfileTasks.test.js:17]: Just nitpicking but should be `task1` :yum:

File: __tests__/components/ProfileTasks.test.js [hunk 2]
```diff
@@ -14,11 +14,15 @@ describe('ProfileTasks', () => {
...
+          betaPopUp: false,
         },
         {
           title: 'Language of correspondence',
           icon: 'stamp',
           link: '/dashboard',
+          areaLabel: 'taks2',
```
Comment (krischarbonneau) [inline __tests__/components/ProfileTasks.test.js:24]: `task2` :yum:

Comment (shewood) [reply __tests__/components/ProfileTasks.test.js]: good catch, typo fixed.

File: __tests__/components/ProfileTasks.test.js (hunk 1 above)
Comment (shewood) [reply __tests__/components/ProfileTasks.test.js]: good catch, typo fixed

**Review by krischarbonneau — APPROVED** [review]
LGTM!

**Pull request merged**